### ISSUE TITLE
Client build

### ISSUE THREAD

**Issue by FRosner** [FRosner/project-pia#6]
``` sh
lein cljsbuild once dev

Retrieving lein-cljsbuild/lein-cljsbuild/1.1.0/lein-cljsbuild-1.1.0.pom from clojars
Retrieving lein-cljsbuild/cljs-compat/1.0.0-SNAPSHOT/cljs-compat-1.0.0-20150824.150307-36.pom from clojars
Retrieving org/clojure/clojure/1.5.1/clojure-1.5.1.pom from central
Retrieving fs/fs/1.1.2/fs-1.1.2.pom from clojars
Retrieving org/clojure/clojure/1.3.0/clojure-1.3.0.pom from central
Retrieving org/apache/commons/commons-compress/1.3/commons-compress-1.3.pom from central
Retrieving org/apache/commons/commons-compress/1.3/commons-compress-1.3.jar from central
Retrieving org/clojure/clojure/1.5.1/clojure-1.5.1.jar from central
Retrieving lein-cljsbuild/cljs-compat/1.0.0-SNAPSHOT/cljs-compat-1.0.0-20150824.150307-36.jar from clojars
Retrieving fs/fs/1.1.2/fs-1.1.2.jar from clojars
Retrieving lein-cljsbuild/lein-cljsbuild/1.1.0/lein-cljsbuild-1.1.0.jar from clojars
Compiling ClojureScript...
Retrieving cljsbuild/cljsbuild/1.1.0/cljsbuild-1.1.0.pom from clojars
Retrieving org/clojure/clojurescript/0.0-2411/clojurescript-0.0-2411.pom from central
Retrieving org/clojure/clojure/1.6.0/clojure-1.6.0.pom from central
Retrieving com/google/javascript/closure-compiler/v20140625/closure-compiler-v20140625.pom from central
Retrieving com/google/javascript/closure-compiler-parent/v20140625/closure-compiler-parent-v20140625.pom from central
Retrieving com/google/javascript/closure-compiler-externs/v20140625/closure-compiler-externs-v20140625.pom from central
Retrieving args4j/args4j/2.0.26/args4j-2.0.26.pom from central
Retrieving args4j/args4j-site/2.0.26/args4j-site-2.0.26.pom from central
Retrieving org/kohsuke/pom/6/pom-6.pom from central
Retrieving com/google/guava/guava/17.0/guava-17.0.pom from central
Retrieving com/google/guava/guava-parent/17.0/guava-parent-17.0.pom from central
Retrieving org/clojure/google-closure-library/0.0-20140718-946a7d39/google-closure-library-0.0-20140718-946a7d39.pom from central
Retrieving org/clojure/google-closure-library-third-party/0.0-20140718-946a7d39/google-closure-library-third-party-0.0-20140718-946a7d39.pom from central
Retrieving org/clojure/data.json/0.2.3/data.json-0.2.3.pom from central
Retrieving org/clojure/pom.contrib/0.1.2/pom.contrib-0.1.2.pom from central
Retrieving org/clojure/clojure/1.4.0/clojure-1.4.0.pom from central
Retrieving org/mozilla/rhino/1.7R4/rhino-1.7R4.pom from central
Retrieving org/clojure/tools.reader/0.8.10/tools.reader-0.8.10.pom from central
Retrieving clj-stacktrace/clj-stacktrace/0.2.5/clj-stacktrace-0.2.5.pom from clojars
Retrieving org/clojure/clojure/1.7.0/clojure-1.7.0.pom from central
Retrieving org/clojure/core.async/0.1.346.0-17112a-alpha/core.async-0.1.346.0-17112a-alpha.pom from central
Retrieving org/clojure/tools.analyzer.jvm/0.1.0-beta12/tools.analyzer.jvm-0.1.0-beta12.pom from central
Retrieving org/clojure/tools.analyzer/0.1.0-beta12/tools.analyzer-0.1.0-beta12.pom from central
Retrieving org/clojure/core.memoize/0.5.6/core.memoize-0.5.6.pom from central
Retrieving org/clojure/core.cache/0.6.3/core.cache-0.6.3.pom from central
Retrieving org/clojure/pom.contrib/0.0.26/pom.contrib-0.0.26.pom from central
Retrieving org/clojure/data.priority-map/0.0.2/data.priority-map-0.0.2.pom from central
Retrieving org/clojure/pom.contrib/0.0.25/pom.contrib-0.0.25.pom from central
Retrieving org/ow2/asm/asm-all/4.1/asm-all-4.1.pom from central
Retrieving org/clojure/clojurescript/1.7.48/clojurescript-1.7.48.pom from central
Retrieving com/google/javascript/closure-compiler/v20150729/closure-compiler-v20150729.pom from central
Retrieving com/google/javascript/closure-compiler-parent/v20150729/closure-compiler-parent-v20150729.pom from central
Retrieving com/google/javascript/closure-compiler-externs/v20150729/closure-compiler-externs-v20150729.pom from central
Retrieving com/google/code/gson/gson/2.2.4/gson-2.2.4.pom from central
Retrieving org/clojure/google-closure-library/0.0-20150805-acd8b553/google-closure-library-0.0-20150805-acd8b553.pom from central
Retrieving org/clojure/google-closure-library-third-party/0.0-20150805-acd8b553/google-closure-library-third-party-0.0-20150805-acd8b553.pom from central
Retrieving org/clojure/data.json/0.2.6/data.json-0.2.6.pom from central
Retrieving org/mozilla/rhino/1.7R5/rhino-1.7R5.pom from central
Retrieving org/clojure/tools.reader/0.10.0-alpha3/tools.reader-0.10.0-alpha3.pom from central
Retrieving cljs-http/cljs-http/0.1.24/cljs-http-0.1.24.pom from clojars
Retrieving com/cemerick/austin/0.1.3/austin-0.1.3.pom from clojars
Retrieving org/clojure/clojurescript/0.0-2014/clojurescript-0.0-2014.pom from central
Retrieving com/google/javascript/closure-compiler/v20130603/closure-compiler-v20130603.pom from central
Retrieving args4j/args4j/2.0.16/args4j-2.0.16.pom from central
Retrieving args4j/args4j-site/2.0.16/args4j-site-2.0.16.pom from central
Retrieving org/clojure/google-closure-library/0.0-20130212-95c19e7f0f5f/google-closure-library-0.0-20130212-95c19e7f0f5f.pom from central
Retrieving org/clojure/google-closure-library-third-party/0.0-20130212-95c19e7f0f5f/google-closure-library-third-party-0.0-20130212-95c19e7f0f5f.pom from central
Retrieving org/clojure/tools.reader/0.7.10/tools.reader-0.7.10.pom from central
Retrieving com/cemerick/piggieback/0.1.2/piggieback-0.1.2.pom from central
Retrieving org/clojure/tools.nrepl/0.2.3/tools.nrepl-0.2.3.pom from central
Retrieving org/clojure/clojure/1.2.0/clojure-1.2.0.pom from central
Retrieving noencore/noencore/0.1.18/noencore-0.1.18.pom from clojars
Retrieving com/cemerick/austin/0.1.5/austin-0.1.5.pom from clojars
Retrieving org/clojure/clojurescript/0.0-2227/clojurescript-0.0-2227.pom from central
Retrieving com/google/javascript/closure-compiler/v20131014/closure-compiler-v20131014.pom from central
Retrieving org/clojure/google-closure-library/0.0-20140226-71326067/google-closure-library-0.0-20140226-71326067.pom from central
Retrieving org/clojure/google-closure-library-third-party/0.0-20140226-71326067/google-closure-library-third-party-0.0-20140226-71326067.pom from central
Retrieving org/clojure/tools.reader/0.8.3/tools.reader-0.8.3.pom from central
Retrieving com/cemerick/piggieback/0.1.3/piggieback-0.1.3.pom from central
Retrieving org/clojure/clojurescript/0.0-2080/clojurescript-0.0-2080.pom from central
Retrieving org/clojure/tools.reader/0.8.0/tools.reader-0.8.0.pom from central
Retrieving org/clojure/clojurescript/0.0-2498/clojurescript-0.0-2498.pom from central
Retrieving com/cognitect/transit-cljs/0.8.194/transit-cljs-0.8.194.pom from central
Retrieving com/cognitect/transit-js/0.8.746/transit-js-0.8.746.pom from central
Retrieving prismatic/schema/1.0.1/schema-1.0.1.pom from clojars
Retrieving prismatic/plumbing/0.5.0/plumbing-0.5.0.pom from clojars
Retrieving de/kotka/lazymap/3.1.0/lazymap-3.1.0.pom from clojars
Retrieving org/omcljs/om/0.9.0/om-0.9.0.pom from clojars
Retrieving cljsjs/react/0.13.3-0/react-0.13.3-0.pom from clojars
Retrieving sablono/sablono/0.3.6/sablono-0.3.6.pom from clojars
Retrieving org/clojure/clojure/1.7.0-beta3/clojure-1.7.0-beta3.pom from central
Retrieving org/clojure/tools.nrepl/0.2.6/tools.nrepl-0.2.6.pom from central
Retrieving clojure-complete/clojure-complete/0.2.3/clojure-complete-0.2.3.pom from clojars
Retrieving org/clojure/clojure/1.7.0/clojure-1.7.0.jar from central
Retrieving org/clojure/core.async/0.1.346.0-17112a-alpha/core.async-0.1.346.0-17112a-alpha.jar from central
Retrieving org/clojure/tools.analyzer.jvm/0.1.0-beta12/tools.analyzer.jvm-0.1.0-beta12.jar from central
Retrieving org/clojure/core.cache/0.6.3/core.cache-0.6.3.jar from central
Retrieving org/clojure/data.priority-map/0.0.2/data.priority-map-0.0.2.jar from central
Retrieving org/clojure/core.memoize/0.5.6/core.memoize-0.5.6.jar from central
Retrieving org/clojure/tools.analyzer/0.1.0-beta12/tools.analyzer-0.1.0-beta12.jar from central
Retrieving org/ow2/asm/asm-all/4.1/asm-all-4.1.jar from central
Retrieving org/clojure/clojurescript/1.7.48/clojurescript-1.7.48.jar from central
Retrieving com/google/javascript/closure-compiler-externs/v20150729/closure-compiler-externs-v20150729.jar from central
Retrieving com/google/javascript/closure-compiler/v20150729/closure-compiler-v20150729.jar from central
Retrieving args4j/args4j/2.0.26/args4j-2.0.26.jar from central
Retrieving com/google/code/gson/gson/2.2.4/gson-2.2.4.jar from central
Retrieving org/clojure/google-closure-library/0.0-20150805-acd8b553/google-closure-library-0.0-20150805-acd8b553.jar from central
Retrieving org/clojure/google-closure-library-third-party/0.0-20150805-acd8b553/google-closure-library-third-party-0.0-20150805-acd8b553.jar from central
Retrieving org/clojure/data.json/0.2.6/data.json-0.2.6.jar from central
Retrieving org/mozilla/rhino/1.7R5/rhino-1.7R5.jar from central
Retrieving org/clojure/tools.reader/0.10.0-alpha3/tools.reader-0.10.0-alpha3.jar from central
Retrieving com/cemerick/piggieback/0.1.2/piggieback-0.1.2.jar from central
Retrieving com/cognitect/transit-cljs/0.8.194/transit-cljs-0.8.194.jar from central
Retrieving com/cognitect/transit-js/0.8.746/transit-js-0.8.746.jar from central
Retrieving org/clojure/tools.nrepl/0.2.6/tools.nrepl-0.2.6.jar from central
Retrieving cljs-http/cljs-http/0.1.24/cljs-http-0.1.24.jar from clojars
Retrieving clj-stacktrace/clj-stacktrace/0.2.5/clj-stacktrace-0.2.5.jar from clojars
Retrieving cljsbuild/cljsbuild/1.1.0/cljsbuild-1.1.0.jar from clojars
Retrieving prismatic/plumbing/0.5.0/plumbing-0.5.0.jar from clojars
Retrieving com/cemerick/austin/0.1.3/austin-0.1.3.jar from clojars
Retrieving prismatic/schema/1.0.1/schema-1.0.1.jar from clojars
Retrieving noencore/noencore/0.1.18/noencore-0.1.18.jar from clojars
Retrieving de/kotka/lazymap/3.1.0/lazymap-3.1.0.jar from clojars
Retrieving org/omcljs/om/0.9.0/om-0.9.0.jar from clojars
Retrieving cljsjs/react/0.13.3-0/react-0.13.3-0.jar from clojars
Retrieving sablono/sablono/0.3.6/sablono-0.3.6.jar from clojars
Retrieving clojure-complete/clojure-complete/0.2.3/clojure-complete-0.2.3.jar from clojars
Compiling "resources/public/js/compiled/dev.js" from ["src/cljs/" "target/generated/src/cljs"]...
WARNING: uuid already refers to: cljs.core/uuid being replaced by: cognitect.transit/uuid at line 318 file:/Users/frosner/.m2/repository/com/cognitect/transit-cljs/0.8.194/transit-cljs-0.8.194.jar!/cognitect/transit.cljs
Compiling "resources/public/js/compiled/dev.js" failed.
java.io.FileNotFoundException: target/generated/src/cljs (No such file or directory)
    at java.io.FileInputStream.open(Native Method)
    at java.io.FileInputStream.<init>(FileInputStream.java:146)
    at clojure.java.io$fn__9189.invoke(io.clj:229)
    at clojure.java.io$fn__9102$G__9095__9109.invoke(io.clj:69)
    at clojure.java.io$fn__9163.invoke(io.clj:165)
    at clojure.java.io$fn__9115$G__9091__9122.invoke(io.clj:69)
    at clojure.java.io$reader.doInvoke(io.clj:102)
    at clojure.lang.RestFn.invoke(RestFn.java:410)
    at cljs.closure$compile_file.invoke(closure.clj:429)
    at cljs.closure$eval3581$fn__3582.invoke(closure.clj:479)
    at cljs.closure$eval3533$fn__3534$G__3524__3541.invoke(closure.clj:383)
    at cljs.closure$eval3594$fn__3595.invoke(closure.clj:492)
    at cljs.closure$eval3533$fn__3534$G__3524__3541.invoke(closure.clj:383)
    at cljsbuild.compiler.SourcePaths$fn__4074.invoke(compiler.clj:67)
    at clojure.core$map$fn__4553.invoke(core.clj:2622)
    at clojure.lang.LazySeq.sval(LazySeq.java:40)
    at clojure.lang.LazySeq.seq(LazySeq.java:49)
    at clojure.lang.RT.seq(RT.java:507)
    at clojure.core$seq__4128.invoke(core.clj:137)
    at clojure.core$apply.invoke(core.clj:630)
    at clojure.core$mapcat.doInvoke(core.clj:2660)
    at clojure.lang.RestFn.invoke(RestFn.java:423)
    at cljsbuild.compiler.SourcePaths._compile(compiler.clj:67)
    at cljs.closure$build.invoke(closure.clj:1671)
    at cljs.closure$build.invoke(closure.clj:1627)
    at cljsbuild.compiler$compile_cljs$fn__4085.invoke(compiler.clj:81)
    at cljsbuild.compiler$compile_cljs.invoke(compiler.clj:80)
    at cljsbuild.compiler$run_compiler.invoke(compiler.clj:187)
    at user$eval4219$iter__4255__4259$fn__4260$fn__4278.invoke(form-init925235253740177075.clj:1)
    at user$eval4219$iter__4255__4259$fn__4260.invoke(form-init925235253740177075.clj:1)
    at clojure.lang.LazySeq.sval(LazySeq.java:40)
    at clojure.lang.LazySeq.seq(LazySeq.java:49)
    at clojure.lang.RT.seq(RT.java:507)
    at clojure.core$seq__4128.invoke(core.clj:137)
    at clojure.core$dorun.invoke(core.clj:3009)
    at clojure.core$doall.invoke(core.clj:3025)
    at user$eval4219.invoke(form-init925235253740177075.clj:1)
    at clojure.lang.Compiler.eval(Compiler.java:6782)
    at clojure.lang.Compiler.eval(Compiler.java:6772)
    at clojure.lang.Compiler.load(Compiler.java:7227)
    at clojure.lang.Compiler.loadFile(Compiler.java:7165)
    at clojure.main$load_script.invoke(main.clj:275)
    at clojure.main$init_opt.invoke(main.clj:280)
    at clojure.main$initialize.invoke(main.clj:308)
    at clojure.main$null_opt.invoke(main.clj:343)
    at clojure.main$main.doInvoke(main.clj:421)
    at clojure.lang.RestFn.invoke(RestFn.java:421)
    at clojure.lang.Var.invoke(Var.java:383)
    at clojure.lang.AFn.applyToHelper(AFn.java:156)
    at clojure.lang.Var.applyTo(Var.java:700)
    at clojure.main.main(main.java:37)
Subprocess failed
```

**Comment by FRosner** [comment]
@Gerrrr any ideas?

**Comment by FRosner** [comment]
Works now, thanks. Fixed by #7
